### DISCUSSION TITLE
Summit Dinner Questions

### DISCUSSION THREAD

**Discussion by sweaglesw** [delph-in/docs#66] [Team Posts]
The 2022 Summit is fast approaching!  I would like to pin down a few remaining local organization details if possible.

1. Everyone is invited to an informal welcome dinner at 7pm the night before the summit begins, at Keenan's Restaurant, which is inside of the Chrysalis (summit venue) hotel.  That is Sunday July 17th.  I would like to have a rough number of guests beforehand to tell the restaurant.  If you expect to be able to join us, please RSVP either here or to me by email.

2. Our Wednesday afternoon excursion cruise includes a crab and chicken dinner.  If you would prefer a vegetarian option, please let me know as soon as possible so that I can be sure to have the right quantity of veggie dishes available.

Thanks! ~Woodley

**Comment by emilymbender** [comment]
Thanks, Woodley! I RSVPed (with some questions) by email. Just checking to make sure you got it :)
